data/tlg0358/tlg001/__cts__.xml

```diff
@@ -1,7 +1,7 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg0358" xml:lang="grc" urn="urn:cts:greekLit:tlg0358.tlg001">
   <ti:title xml:lang="lat">Introductio arithmetica</ti:title>
-  <ti:edition urn="urn:cts:greekLit:tlg0358.tlg001.1st1K-grc1" workUrn="urn:cts:greekLit:tlg0358.tlg001">
+  <ti:translation  xml:lang="eng"  urn="urn:cts:greekLit:tlg0358.tlg001.1st1K-grc1" workUrn="urn:cts:greekLit:tlg0358.tlg001">
```

Comment:
I don't think this file is actually related to this pull request. But in any case, since this is a Greek text of a Greek original, I think it should be `<ti:edition>` instead of `<ti:translation>`.